scripts/compatibility_matrix.py

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env python3
+
+# License: CC0 https://creativecommons.org/share-your-work/public-domain/cc0/
+
+# A messy script that figures out ZFS features. It's very messy, sorry. I am
+# not responsible if this script eats your laundry.
+#
+# This uses manpages, because I'm lazy. If your manpages are wrong, you have a
+# bug.
+#
+# If the script is wrong, or could be improved, feel free to contact me on
+# freenode, and tell me why it's wrong. My nick is zgrep.
+#
+# 2018-07-05: Created.
+# ????-??-??: Many things happened.
+# 2020-02-23: Applied patch by rlaager.
+# 2020-02-30: Show domain prefixes (via Vlad Bokov), partially apply patch by
+#   rlaager.
+# 2020-03-05: Patch by rlaager (allocation_classes, ZoL -> openzfs).
+# 2020-10-07: Rework by gmelikov for openzfs-docs
```

Comment:
I'm not sure that the history is particularly critical once this is merged in. We'll have the git history.

Reply:
Cleaned header info totally, add basic description.